src/Uno.UI/UI/Xaml/Controls/TimePicker/UnoTimePickerDialog.Android.cs

```diff
@@ -88,7 +89,7 @@ private void SetMinuteIncrement()
 				minutePicker.Value = values.FindIndex(num => num == _minute);
 				minutePicker.MinValue = 0;
 				minutePicker.MaxValue = values.Count - 1;
-				minutePicker.SetDisplayedValues(values.Select(num => num.ToString("00")).ToArray());
+				minutePicker.SetDisplayedValues(values.Select(num => num.ToString("00", CultureInfo.InvariantCulture)).ToArray());
```

Comment:
`CurrentUICulture` this is presented to the user.

Reply:
```suggestion
				minutePicker.SetDisplayedValues(values.Select(num => num.ToString("00", CultureInfo.CurrentUICulture)).ToArray());
```